### PR TITLE
Sprint 25 odim 3323

### PR DESCRIPTION
**Pull request by amar-shalgar**
Added create volume unit tests for svc-api

### CODE REVIEW

**Review by a-ajith — APPROVED** [review]

File: svc-api/router/router.go [hunk 1]
```diff
@@ -219,6 +220,7 @@ func Router() *iris.Application {
 	systems.Get("/{id}/Storage/{rid}", system.GetSystemResource)
 	systems.Get("/{id}/Storage/{rid}/Drives", system.GetSystemResource)
```
Comment (jeevan-kamkar) [inline svc-api/router/router.go]: create a route party as storage under this add these uri

File: svc-api/handle/systems_test.go [hunk 2]
```diff
@@ -469,3 +469,94 @@ func TestSetDefaultBootOrderWithValidData(t *testing.T) {
...
+	var sys SystemRPCs
+	sys.CreateVolumeRPC = mockCreateVolume
+	mockApp := iris.New()
+	redfishRoutes := mockApp.Party("/redfish/v1/Systems")
+	redfishRoutes.Post("/{id}/Storage/{rid}/Volumes", sys.CreateVolume)
+
+	e := httptest.New(t, mockApp)
+	e.POST(
+		"/redfish/v1/Systems/6d4a0a66-7efa-578e-83cf-44dc68d2874e:1/Storage/ArrayControllers-0/Volumes",
+	).WithJSON(map[string]string{"Sample": "Body"}).WithHeader("X-Auth-Token", "ValidToken").Expect().Status(http.StatusOK)
```
Comment (jeevan-kamkar) [inline svc-api/handle/systems_test.go]: change expected status code StatusNotImplemented and  add to comments change the status code with actual status code and after code is added update the status code

File: svc-api/handle/systems.go [hunk 3]
```diff
@@ -312,3 +313,10 @@ func (sys *SystemRPCs) ChangeBootOrderSettings(ctx iris.Context) {
 	ctx.StatusCode(int(resp.StatusCode))
 	ctx.Write(resp.Body)
 }
+
+// CreateVolume is the handler to create a volume under storage
+// from iris context will get the request and check sessiontoken
+// and do rpc call and send response back
+func (sys *SystemRPCs) CreateVolume(ctx iris.Context) {
+	// This function yet to be implemented
```
Comment (jeevan-kamkar) [inline svc-api/handle/systems.go:321]: retrun StatusNotImplemented status   after code is added remove that

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Change status code in ut to the Status Not Implemented and to do comment to update the same in all occurrences

File: svc-api/router/router.go (hunk 1 above)
Comment (amar-shalgar) [reply svc-api/router/router.go]: Addressed the comment

File: svc-api/handle/systems_test.go (hunk 2 above)
Comment (amar-shalgar) [reply svc-api/handle/systems_test.go]: Addressed the comment

File: svc-api/handle/systems.go (hunk 3 above)
Comment (amar-shalgar) [reply svc-api/handle/systems.go:321]: Addressed the comment

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**